storage_service/locations/models/arkivum.py

```diff
@@ -207,6 +200,39 @@ def _get_package_info(self, package):
             return {'error': True, 'error_message': msg}
         return response_json
 
+    def _get_baginfo_txt_info(self, package):
+        """
+        Return information for the bag-info.txt from a package in Arkivum.
+        To be used as a fallback to obtain replication status of an
+        uncompressed package when it is not returned by the
+        ingest-manifest/status call in _get_package_info()
+        """
+        location = Location.objects.get(uuid=package.current_location_id)
+        url = 'https://' + self.host + '/api/2/files/fileInfo/' + \
+            location.relative_path + '/' + \
+            package.current_path + '/bag-info.txt'
+        LOGGER.info('URL: %s', url)
```

Comment:
I have confirmed that `location = Location.objects.get(uuid=package.current_location_id)` can be replaced with the simpler `location = package.current_location`, which may not incur an additional db query.

Also, I think we may want to strip leading and trailing forward slashes from the variables in the URL in order to avoid double forward slashes. Something like the following should work:

    url = 'https://' + self.host + '/api/2/files/fileInfo/' + \
        location.relative_path.strip('/') + '/' + \
        package.current_path.strip('/') + '/bag-info.txt'